spec/models/user_spec.rb

```diff
@@ -4,6 +4,12 @@
 MAX_GOOD_PASSWORD = '!1aZ' * 32
 
 describe User do
+  describe 'Associations' do
+    it { should have_many(:authorizations) }
```

Comment:
I was thrown off by the `should` when I first saw it as well, but I have since discovered it is part of https://github.com/thoughtbot/shoulda-matchers gem and not an invocation of the older rspec syntax.